dotcom-rendering/src/components/MiniProfile.tsx

```diff
@@ -0,0 +1,186 @@
+import { css } from '@emotion/react';
+import { type ArticleFormat } from '@guardian/libs';
+import { neutral, textSans14 } from '@guardian/source/foundations';
+import type { EditionId } from '../lib/edition';
+import type { ArticleElementRenderer } from '../lib/renderElement';
+import { slugify } from '../model/enhance-H2s';
+import { sanitiseHTML } from '../model/sanitise';
```

Comment:
We actually [use `sanitize-html` thrice](https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering+%22from+%27sanitize-html%27%22&type=code) and [`sanitise.ts`’s sanitiseHTML only once](https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering+%22+sanitiseHTML+%22&type=code).

Reply:
I will follow the wisdom of the crowd.

Reply:
(using `sanitise` from `sanitise-html` now)